cms/jinja2/templates/datavis/_macro.njk

```diff
@@ -33,5 +31,15 @@
                 {% endif %}
             </ol>
         {% endif %}
+
+        <!-- script tags to pass json data to the javascript component -->
+        <script type="application/json" data-highcharts-config--{{ params.uuid }}>
```

Comment:
Suggestion: Rather than customizing the data-attribute itself, perhaps worth using a value? `data-highcharts="{{ params.uuid }}"`.

Reply:
good idea - now done.